doc/developer-guides/hld/hv-rdt.rst

```diff
@@ -0,0 +1,54 @@
+.. _hv_rdt:
+
+RDT Allocation Feature Supported by Hypervisor
+##############################################
+
+The hypervisor allows to use RDT (Resource Director Technology) allocation features to optimize performances of VMs. There are 2 sub-features: CAT (Cache Allocation Technology) and MBA(Memory Bandwidth Allocation), CAT is for cache resources and MBA is for memory bandwidth resources. The RDT in hypervisor is integrated with VM-Configuration feature, the resources consumed by each VM can be configured and restricted at compile time.
```

Comment:
In ACRN, the CAT is configured via the "VM-Configuration", the resources allocated for VMs are determined in the VM configuration.

Reply:
Use it!